routers/chats/utilities/summary.py

```
@@ -1,11 +1,12 @@
+from datetime import datetime
```

Ruff (0.8.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)